website/docs/api/Browser.md

```diff
@@ -9,7 +9,9 @@ The browser object is the session instance you use to control the browser or mob
 
 The session is initialized by the test runner. The same goes for ending the session. This is also done by the test runner process.
 
-Besides all commands from the [API](API.md), the `browser` object provides some more information you might be interested in during your test run:
+The browser object provides all methods from the [communication protocol](Protocols.md) section, e.g. [Chromium](/docs/api/chromium) communication protocol (if you are using Chromedriver) as well as methods listed within this section. Available methods depend on the type of session. If you run an Chrome test session, none of the Appium [methods](/docs/api/appium) will be available on the browser object and vice versa.
```

Comment:
I don't think highlighting Chromium makes sense here. Can we paraphrase this more general, saying that "the browser object provides [protocol methods](/docs/api/protocols) based on the detected WebDriver backend".

Reply:
I don't have particular interest with Chromium either. But the package name is WebdriverIO, the example Webdriver, which are confusing, hence since its just an example, we can take anything other than "webdriver" to make it simpler and easier for quick documentation reading. Please note people will not read all letters of the word to comprehend the word.

Chromium is common knowledge, and webdriver backend etc., again will require some pre-requisite knowledge, that its a std, protocol, etc.etc. Chromium is understood as browser, for understanding webdriver users have to reasearch what is webdriver, etc. etc. which is not essential to use WDIO package.

The majority users are here to just install the WDIO and run tests, they hardly (and rightly) do not care about the technologies that WDIO is using to accomplish its tasks.

I understand you are your intention of being accurate in documenting the code, but its not required for majority audiences to know what they don't need to know for just running a test. (WDIO should automatically take care of the rest, and it does). Majority audience are here to just use the package as testing facilitator.

These are my inputs, please do feel free to force changes or reject the PR.